starter-code/functions-and-arrays.js

```diff
@@ -20,67 +20,117 @@ var words = [
   "orchard",
   "crackpot"
 ];
+
+function findLongestWord(words) {
```

Comment:
This works fine. However you don't need `num`because you can simply to `long.length`.